docs/Actions/execute-token.md

```diff
@@ -0,0 +1,130 @@
+---
+id: execute-token
+title: Execute Token

```

Comment:
Apply feedback from apply-tokens to this action as well.

Reply:
done